tcms/bugs/templates/bugs/get.html

```diff
@@ -134,6 +134,30 @@ <h2 class="card-pf-title">
         </div>
     </div>
     {% endif %}
+
+    {% if perms.django_comments.add_comment and not object.status %}
+    <div class="row row-cards-pf">
+        <div class="col-xs-12 col-sm-12 col-md-12">
+            <div class="card-pf">
+                <form class="form-horizontal" action="{% url 'bugs-comment' %}" method="post">
+                    <input name="bug" type="hidden" value="{{ object.pk }}">
+                    <div class="form-group">
+                        <div>
+                            <div>{{ comment_form.text }}</div>
+                            {{ comment_form.text.errors }}
+                        </div>
+                    </div>
+
+                    <div class="form-group">
+                        <div>
+                            <button type="submit" name="action" value="reopen" class="btn btn-warning btn-lg" style="float:right">{% trans "Reopen" %}</button>
```

Comment:
this whole block is copy-pasted from above and this introduces code duplication which we can avoid. what I am thinking is that we can merge the common parts and only if-else the text of the button and value of the comment. something like this:
```html
{% if perms.django_comments.add_comment %}
    <div class="row row-cards-pf">
        ...
                    <div class="form-group">
                        <div>
                            <button type="submit" name="action" value="{% if object.status %}reopen{% else%}close" class="btn btn-warning btn-lg" style="float:right">{% if object.status %}{% trans "Reopen" %}{% else%}{% trans "Close" %}</button>
                        </div>
     ...
   </div>
    {% endif %}
```

Reply:
Yes but is it required to keep the save button for closed bugs.? I guess **bug.status** check should also cover the save button.

Reply:
Please check #1514